src/main/java/frc/robot/Constants.java

```diff
@@ -89,9 +92,18 @@ public static final class AutonConstants {
   }
 
   public static final class VisionConstants {
-    public static final Transform3d FRONT_LEFT_CAMERA_TO_ROBOT = new Transform3d();
-    public static final Transform3d FRONT_RIGHT_CAMERA_TO_ROBOT = new Transform3d();
-    public static final Transform3d REAR_CAMERA_TO_ROBOT = new Transform3d();
+    public static final Transform3d ROBOT_TO_FRONT_LEFT_CAMERA =
+        new Transform3d(
+            new Translation3d(Units.inchesToMeters(9.766), Units.inchesToMeters(7.852), Units.inchesToMeters(28.242)),
+            new Rotation3d());
+    public static final Transform3d ROBOT_TO_FRONT_RIGHT_CAMERA =
+        new Transform3d(
+            new Translation3d(Units.inchesToMeters(9.766), -Units.inchesToMeters(7.852), Units.inchesToMeters(28.242)),
+            new Rotation3d());
+    public static final Transform3d ROBOT_TO_REAR_CAMERA =
+        new Transform3d(new Translation3d(0, -Units.inchesToMeters(12.1585), Units.inchesToMeters(20.749)),
+            new Rotation3d(0, 0, Math.PI));
+    public static final double MAINTAIN_CAMERA_THRESHOLD = 0.5;
```

Comment:
What does this mean? Is there a unit for this?

If this is the confidence threshold, it should probably be higher than 50%

Reply:
yeah it's confidence threshold